cmds/coredhcp-generator/coredhcp.go.template

```diff
@@ -21,9 +23,26 @@ import (
 var (
 	flagLogFile     = flag.String("logfile", "", "Name of the log file to append to. Default: stdout/stderr only")
 	flagLogNoStdout = flag.Bool("nostdout", false, "Disable logging to stdout/stderr")
-	flagDebug       = flag.Bool("debug", false, "Enable debug output")
+	flagLogLevel    = flag.String("loglevel", "info", fmt.Sprintf("Log level. One of %v", getLogLevels()))
 )
 
+var logLevels = map[string]func(*logrus.Logger){
+        "none":    func(l *logrus.Logger) { l.SetOutput(ioutil.Discard) },
```

Comment:
Do we have a log level that avoids any output ?

With this, we'll emit (and format, etc) the log messages, then discard them. With a high level we can prevent the messages from even being evaluated

Reply:
not that I could find - the code would be more compact if it did...